.github/workflows/ci.yml

```diff
@@ -30,11 +30,11 @@ jobs:
 
       - name: Format
         if: runner.os == 'Linux'
-        run: make fmt/check
+        run: deno check
```

Comment:
This is not the same. `deno fmt --check` != `deno check`

The latter is for type checking.